starters/nuxt3-pinia-vuetify/components/CounterExample/CounterExample.vue

```diff
@@ -0,0 +1,61 @@
+<script lang="ts" setup>
+import { useCounterStore } from '@/store/counterStore';
+
+const counterState = useCounterStore();
+
+const increaseCount = () => counterState.counter++;
```

Comment:
Suggested renames to make it even more consistent with the html part of the component:
- Rename `increaseCount` to `increment`
- Rename `decreaseCount` to `decrement`
- `resetCount` is well named, can stay as is :)